src/backend/interfaces/NATS/py.client/caliopen_nats/subscribers.py

```diff
@@ -55,10 +58,23 @@ def handler(self, msg):
 class ContactAction(BaseHandler):
     """Handler for contact action message."""
 
+    def process_update(self, msg, payload):
+        """Process a contact update message."""
+        # XXX validate payload structure
+        if 'user_id' not in payload or 'contact_id' not in payload:
+            raise Exception('Invalid contact_update structure')
+        user = User.get(payload['user_id'])
+        contact = Contact(user.user_id, contact_id=payload['contact_id'])
+        contact.get_db()
```

Comment:
do not forget to `contact.unmarshall_db()` before processing, do you ?

Reply:
we could simply add `methods retrieve_db()` and `retrieve_index()` that do `get_xxx()` then `unmarshall_xxx()` under the hood.

Reply:
that's another story, we will define such interfaces later